libs/dh/esett/feature-balance-responsible/src/lib/dh-balance-responsible.component.ts

```diff
@@ -133,6 +132,7 @@ export class DhBalanceResponsibleComponent implements OnInit {
 
         this.tableDataSource.data = result.data?.balanceResponsible.page ?? [];
         this.totalCount = result.data?.balanceResponsible.totalCount ?? 0;
+        this.url = result.data?.balanceResponsible.balanceResponsiblesUrl ?? '';
```

Comment:
nitpick: even though the `balanceResponsiblesUrl` property can be `null`/`undefined`, MSW complains that it can't find it in the mock.

Here's an error about `storageDocumentUrl`. I assume the same error will show up about `balanceResponsiblesUrl`.

![image](https://github.com/user-attachments/assets/96581400-c1f5-4a0a-8884-a0068a58103a)

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/3419/commits/90648efa75b7385ebc4e1a0cbddf575324016d6d